### PR TITLE
MYST-577 Fix missing version during "dev" builds

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,18 +38,25 @@ script:
 
 before_deploy:
   - bin/travis_scripts/ensure_xgo.sh
-  - bin/package_all $VERSION
+  - bin/package_all $(cat VERSION)
```
Comment (zolia) [inline .travis.yml]: do we need '$' before (cat VERSION) ?

Comment (zolia) [reply .travis.yml]: should `cat VERSION` work?

Comment (Waldz) [reply .travis.yml]: Approach now is different

File: metadata/version.go [hunk 2]
```diff
@@ -42,6 +29,21 @@ const versionSummaryFormat = `Mysterium Node
 %s
 %s`
 
+// VersionAsString returns all defined version constants as single string
+func VersionAsString() string {
+	if Version != "" {
+		return Version
+	}
+
+	version := "source"
```
Comment (zolia) [inline metadata/version.go]: maybe space after 'source'?

Comment (Waldz) [reply metadata/version.go]: Fixed

**Pull request merged**